app/models/concerns/reviews/scopes.rb

```diff
@@ -90,7 +91,8 @@ def list_all_without_final_review_by_date from_date, to_date
         ConclusionReview.table_name => { review_id: nil }
       ).order(
         without_final_review_order
-      ).references(:conclusion_reviews, :business_unit_types)
+      ).allowed_by_business_units.
+      references(:conclusion_reviews, :business_unit_types)
```

Comment:
```suggestion
      ).
      allowed_by_business_units.
      references(:conclusion_reviews, :business_unit_types)
```

Aunque quede raro, un poco menos raro que lo otro.